example/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -11,4 +11,8 @@
           tools:ignore="AcceptsUserCertificates" />
     </trust-anchors>
   </base-config>
+  <domain-config cleartextTrafficPermitted="true">
+    <domain includeSubdomains="true">localhost</domain>
+    <domain includeSubdomains="true">10.0.1.1</domain>
+  </domain-config>
```

Comment:
This seems to be required for our sample app to connect to the Metro bundler in a debug build now. Perhaps something in the newest Android version is blocking this traffic now by default. https://stackoverflow.com/a/70736203